cve_bin_tool/checkers/darkhttpd.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2022 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for darkhttpd
+
+https://www.cvedetails.com/product/112161/Darkhttpd-Project-Darkhttpd.html?vendor_id=26797
+
```

Comment:
```suggestion
Note: darkhttpd is not provided on debian and openWRT. Tests use fedora packages only.
```

Feel free to use alternate wording, I just want to keep the info within the checker so no one has to walk old PRs to find out why only fedora tests were used.  We could also store it in the tests, but I feel like humans are more likely to read checker files than test ones.